marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/main.sjs

```diff
@@ -164,7 +168,14 @@ function buildEnvelope(entityInfo, doc, instance, outputFormat, options) {
       if (attachments instanceof Document && attachments.documentFormat === 'JSON') {
         nb.addText(xdmp.quote(attachments));
       } else {
-        nb.addNode(attachments);
+        // can get sequence of nodes in JSON to XML scenario
```

Comment:
Is this being included to address a separate issue?

Reply:
There isn't a separate issue filed that I've seen, but the unit test would fail without either this change or manipulating the JSON object to have a single root property.